images/mod-tile/README.md

```diff
@@ -29,6 +29,13 @@ The sequence number of last rendered replication will be stated in `/mnt/expired
 - `MOD_TILE_PATH` e.g `hot`
 - `REPLICATION_DIR` the inner directory path for mounted replications
```

Comment:
REPLICATION_DIR is not used by the script.

Reply:
done